### PR TITLE
Use mebibytes over hybrid mega/mebi

### PR DESCRIPTION
**Pull request by inlined**
V1 always used "Mebibytes" (base 2) but used the term "Megabytes" (ambiguous base 2 or 10) for a unitless "availableMemoryMb". In v2, the available memory field has units. In those units, "M" is base 10 and "Mi" is base 2.

This corrects the previous erroneous behavior where we had base 2 quantities and base 10 units.

### CODE REVIEW

File: src/gcp/cloudfunctionsv2.ts [hunk 1]
```diff
@@ -386,6 +386,9 @@ export async function deleteFunction(cloudFunction: string): Promise<Operation>
   }
 }
 
+/**
+ *
+ */
```
Comment (colerogers) [inline src/gcp/cloudfunctionsv2.ts:391]: Empty ts doc comment

File: src/gcp/cloudfunctionsv2.ts [hunk 2]
```diff
@@ -507,6 +516,9 @@ export function functionFromEndpoint(endpoint: backend.Endpoint, source: Storage
   return gcfFunction;
 }
 
+/**
+ *
+ */
```
Comment (colerogers) [inline src/gcp/cloudfunctionsv2.ts:521]: Empty ts doc comment

**Review by colerogers — CHANGES_REQUESTED** [review]
lgtm, delete the empty comments before merge

**Review by inlined — COMMENTED** [review]
Yeah, I don't know why my lint fixer is adding empty comments. Fixed.

**Review by colerogers — APPROVED** [review]

**Pull request merged**